web-animations-2/Overview.bs

```diff
@@ -444,6 +460,8 @@ The procedure to <dfn>set the current time</dfn> of an animation,
 1.  If <var>animation</var> has a <a>pending pause task</a>, synchronously
     complete the pause operation by performing the following steps:
     1.  Set <var>animation</var>'s <a>hold time</a> to <var>seek time</var>.
+    1.  Set <var>animation</var>'s <a>hold phase</a> to
+        [=timeline active phase|active=].
```

Comment:
I think overrides is the wrong term here since hold phase and effect phase are different things.  The effect phase depends on timeline phase and local time.  By setting a hold phase when the animation is paused, we are locking down the timeline phase at the time the animation was paused. Since local time will also be locked down via hold time, we should get a consistent result for the effect change.  

But I do think the hold phase is being set a bit naively here.  Though you can't manually set the scroll position past 100% by dragging a scorllbar, you can programmatically set currentTime to CSS.percent(200), and we should be computing the timeline phase as opposed to making the assumption that it is in bounds.